webdriver-spec.html

```diff
@@ -434,11 +434,15 @@
     <!-- queue --> <li><dfn><a href=https://infra.spec.whatwg.org/#queues>queue</a></dfn>
    </ul>
 
- <dt>Page Visibility
+ <dt>Performance
```

Comment:
The Page Visibility spec is part of the Web Performance WG, but arguably this header should maybe say _Interaction_ since that’s the domain it represents.

Cf. the headers of the other sections: they’re not grouped by the name of the specification but by what general area/domain they are in.